packages/dev/core/src/Gizmos/boundingBoxGizmo.ts

```diff
@@ -182,6 +184,10 @@ export class BoundingBoxGizmo extends Gizmo implements IBoundingBoxGizmo {
      * Fired when a rotation anchor or scale box is dragged
      */
     public onDragStartObservable = new Observable<{}>();
+    /**
+     * Fired when a scale box or a rotation anchor is hovered
+     */
+    public onBoxOrAnchorHoverObservable = new Observable<{}>();

```

Comment:
This should be cleared in dispose

Reply:
Should I add the other observables to dispose as well? I don't see any of the current ones being disposed.

    onDragStartObservable: Observable<{}>;
    onBoxOrAnchorHoverObservable: Observable<{}>;
    onScaleBoxDragObservable: Observable<{}>;
    onScaleBoxDragEndObservable: Observable<{}>;
    onRotationSphereDragObservable: Observable<{}>;
    onRotationSphereDragEndObservable: Observable<{}>;

Reply:
added on newest commit for all observables

Reply:
Thanks a lot !!!